.env

```diff
@@ -1 +1,5 @@
 SERVER_PORT=4550
```

Comment:
Most likely this .env file would not be required and values would only be required to be pushed in from the Command Line.Let this be as such for now and we can see what Dev Ops require going forward.